README.md

```diff
@@ -17,12 +17,26 @@ Python packages with our recommended tooling set up and ready to go.
 > package template (e.g., [`scikit-hep`](https://github.com/scikit-hep/cookie)),
 > we recommend using their template instead of this one.
 
+## Scope and contributing
+
+This project was created by [research software engineers] at [UCL's Centre for Advanced Research Computing][UCL ARC].
+We made it with our own research software projects in mind - we don't explicitly support external users, but whoever you are, we hope you might find it useful!
```

Comment:
I also don't really like "we don't explicitly support external users".

I'm biased to how I had it, but I did spend effort thinking carefully about how it was worded.

Reply:
I like the current one. Whatever happens this should be the "groups" view, not one individuals.